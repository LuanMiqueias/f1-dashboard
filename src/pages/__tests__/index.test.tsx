import { render } from "@testing-library/react";
import Home from "../index";

it("Should math snapshot", () => {
  const { container } = render(<Home />);

  expect(container.firstChild).toMatchSnapshot();
});
