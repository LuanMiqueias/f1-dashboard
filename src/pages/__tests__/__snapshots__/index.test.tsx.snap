// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should math snapshot 1`] = `
<div
  class="container"
>
  <main
    class="main"
  >
    <div
      class="content"
    >
      <h1
        class="title"
      >
        F1 
        <span>
          Dashboard
        </span>
      </h1>
      <p
        class="sub_title"
      >
        Acompanhe seus jogadores preferidos
      </p>
      <button
        class="btn_goDashboard"
      >
        Go to dashboard
      </button>
    </div>
  </main>
  <footer
    class="footer"
  >
    Desenvlvido por
     
    <a
      href="https://luanmiqueias.com.br"
      target="blank"
    >
      Luan Miqueias
    </a>
  </footer>
</div>
`;
